### ISSUE TITLE
Add example return model.in readme

### ISSUE THREAD

**Issue by PcolBP** [s0nerik/fast_contacts#5]
Hi, thanks for package. Anyway, we are thinking now to use your package in our project, we have to add and test it first. 

Would be great to see which data is returned from reading and how it looks - structure.

That would help others to see if there is missing any data that they need without need for implementing it first.

Thanks in advance.

**Comment by s0nerik** [comment]
Hey, @PcolBP , there's already a [description of what info is available for a contact](https://github.com/s0nerik/fast_contacts#available-contact-info) in README, but you can look at the actual contact model [here](https://github.com/s0nerik/fast_contacts/blob/8420969ea252c197078b02f4ce79e6cabf4cb33e/lib/fast_contacts.dart#L7).

**Comment by PcolBP** [comment]
Nice, thank you very much. 

Ill take opportunity to ask you one more thing. Can I run your code in separate isolate? I have tried it with flutter_contacts but that package need bindings to run, unfortunately sub threads are just simple dart isolates which doesnt support flutter widgets and binding.

Do you know or havent tried yet? Let me know, thank you once again for your time.

**Comment by s0nerik** [comment]
I haven't tried running this plugin on a separate isolate, but I'm pretty sure it won't work out of the box as it relies on method channels, which require an isolate setup.

Anyway, wrapping the contacts fetching in an isolate wouldn't speed up anything. If you really need to access fetched contacts on a separate isolate for some reason, it should be possible to achieve that using port-based communication between the main isolate and yours.

**Comment by PcolBP** [comment]
My main goal is to not freeze ui while fetching data. I have client which have over 11k contacts in phone and that may result in freeze ui anyway. 

I found out that there is plugin  https://pub.dev/packages/flutter_isolate which creates flutter engine in isolate so there is a way to call platform channel.

Thanks for your time and answers.

I whish you nice weekend and happy incoming new year!

**Comment by s0nerik** [comment]
Yeah, 11k contacts will require 11k allocations on the main isolate, and that certainly can freeze the UI a little. Wrapping it in an isolate will certainly fix the UI freeze if you use the contacts within that same isolate. If you plan on returning them to the main one - copying will occur and I fear it might take some noticeable UI isolate time as well.

I think this situation can be improved by making the returned contacts iteration lazier. I think it might be feasible to add something like `FastContacts.allContactsLazy` that would return an `Iterable<Contact>`, which allocates a contact only when it is requested. This way, you'll be able to delay the processing of the next chunk to the next frame, "smearing" the total allocation costs between the frames.

**Comment by PcolBP** [comment]
As far as I know currently in flutter isolates dont copy data from one to another thread which shouldn't fresze ui while allocating data from one thread to another.

But yeah, lazy would do the thing.

**Comment by PcolBP** [comment]
Hi @s0nerik, I was able to run your plugin in isolates thanks to [flutter_isolate](https://pub.dev/packages/flutter_isolate).

Unfortunately I couldn't use your functions since we can not return custom dart models between isolates. So I just invoked your function within MethodChannel. Everything work as expected.

Anyway, topic was about model in readme, and since you replied to me and answered, I feel satisfied. Thanks once again for plugin and your time 👍 

```dart 

@pragma('vm:entry-point')
  static Future<List<Map<String, dynamic>>> getContactsFromDevice(
    int code,
  ) async {
    final rawContacts =
        await const MethodChannel('com.github.s0nerik.fast_contacts')
            .invokeMethod<List>('getContacts');
    return rawContacts
            ?.map(
              (raw) => jsonDecode(jsonEncode(raw)) as Map<String, dynamic>,
            )
            .toList() ??
        [];
  }
  
```


Feel free to close issue if you want to.

**Comment by PcolBP** [comment]
Hi @s0nerik, we have found a small issue? I guess. In your [link](https://github.com/s0nerik/fast_contacts/blob/8420969ea252c197078b02f4ce79e6cabf4cb33e/lib/fast_contacts.dart#L7) which you applied to me there is Organization model but inside plugin there is no organization image below. First one is from our code, the second one is from your link. 
![image](https://user-images.githubusercontent.com/67896443/208652918-92f0097b-4465-4d3e-9189-2eb371634c50.png)
![image](https://user-images.githubusercontent.com/67896443/208652970-4755d744-68db-4d07-8eb3-aa4c5e52090d.png)

Did you forget to upload changes? Let me know :) Im using the latest version 1.1.1

**Comment by s0nerik** [comment]
You're totally correct! I forgot to publish the new version after merging https://github.com/s0nerik/fast_contacts/pull/4 🤦 I'll do it soon.

**Comment by s0nerik** [comment]
BTW, [PcolBP](https://github.com/PcolBP), I've got curious about the main isolate freeze upon receiving a large contacts list. What I found out is that the actual culprit is a synchronous decoding of the data returned from the method channel within `MethodChannel.invokeMethod()`. So I tried baking the `flutter_isolate` right into the plugin, but found out that in order to return a `List<Contact>` from another isolate to the main one, decoding on the main isolate would still be required, making this approach basically the same (or even worse) performance-wise as not using the isolate at all.

Have you somehow managed to decode contacts on the main isolate efficiently?
